Ginger/GingerCoreCommon/ReporterLib/StatusMsgsPool.cs

```diff
@@ -71,6 +72,7 @@ public static void LoadStatusMsgsPool()
             Reporter.StatusMsgsPool.Add(eStatusMsgKey.RecommendNewVersion, new StatusMsg(eStatusMsgType.INFO, "Upgrade Required", "Newer version of Ginger exist." + System.Environment.NewLine + "You can download the latest version from https://ginger.amdocs.com/", true, "Upgrade"));
             Reporter.StatusMsgsPool.Add(eStatusMsgKey.NewVersionAvailable, new StatusMsg(eStatusMsgType.INFO, "New version ({0}) is available", "Newer version of Ginger is available." + System.Environment.NewLine + "You can download the latest version from https://ginger.amdocs.com/", true, "Upgrade"));
             Reporter.StatusMsgsPool.Add(eStatusMsgKey.SaveItem, new StatusMsg(eStatusMsgType.PROCESS, "Saving", "Saving '{0}' {1}"));
+            Reporter.StatusMsgsPool.Add(eStatusMsgKey.UpdateItem, new StatusMsg(eStatusMsgType.PROCESS, "Updating", "Updating all the references of '{0}' and replacing with '{1}'"));
```

Comment:
Change key to RenameItem and message to "Renaming all references of  <<old name>> to <<New name>>"